lib/zendesk_api/resources.rb

```diff
@@ -796,6 +796,24 @@ class GreetingCategory < Resource
     class Ticket < CreateResource
       namespace "channels/voice"
     end
+
+    class Agent < ReadResource
+      namespace "channels/voice"
+
+      class Ticket < CreateResource
+        def new_record?
+          true
+        end
+
+        def self.display(client, options)
```

Comment:
Could this be an instance method on Ticket? i.e. `Ticket.new(....).display!`. I'm also okay with this way, just call it `self.display!`.